spec/api/v1/trade_leads_spec.rb

```diff
@@ -25,7 +26,7 @@
 
       it 'returns trade leads sorted by published_date:desc, country: asc' do
         json_response = JSON.parse(response.body)
-        expect(json_response['total']).to eq(13)
+        expect(json_response['total']).to eq(16)
```

Comment:
I think this is a side-effect of the fact that we test most of our search functionality through the request specs rather than the Query specs (like I mentioned earlier). I'd rather we move detailed testing of search into Query specs, then just have less documents imported for request specs, so that we don't even need to think about problems like this. But that's a refactoring for another day, not part of this story.